src/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/sdk-scheme-adapter",
-  "version": "9.4.11",
+  "version": "9.4.12",
```

Comment:
Is this not 10.1.x?

Reply:
Thank you @sri-miriyala it should be :-) .. Here's the schedule for future reference: https://github.com/mojaloop/documentation/blob/master/contributors-guide/standards/versioning.md#sprint-schedule-for-pi0

Reply:
```suggestion
  "version": "10.1.0",
```